src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -660,7 +963,7 @@ StaticLLMPipeline::StaticLLMPipeline(
     const auto use_blobs = pop_or_default(properties, "USE_BLOBS", false);
     if (!use_blobs) {
         ModelConfigDesc model_desc = get_modeldesc_from_json(models_path / "config.json");
-        auto model = genai::utils::singleton_core().read_model(models_path / "openvino_model.xml", {}, properties);
```

Comment:
@ilya-lavrenov, it was changed by you last time:
https://github.com/openvinotoolkit/openvino.genai/blame/master/src/cpp/src/llm_pipeline_static.cpp#L663

Reply:
yes, please, keep master's branch version of `read_model`